diff -ruN libcxxabi-14.0.0.src.orig/CMakeLists.txt libcxxabi-14.0.0.src/CMakeLists.txt
--- a/libcxxabi-14.0.0.src.orig/CMakeLists.txt	2022-03-24 08:12:22.531338455 +0100
+++ a/libcxxabi-14.0.0.src/CMakeLists.txt	2022-03-24 08:13:21.947731822 +0100
@@ -46,6 +46,7 @@
 include(GNUInstallDirs)
 
 if (LIBCXXABI_STANDALONE_BUILD)
+  find_package(LLVM REQUIRED HINTS "${LLVM_CMAKE_DIR}")
 
   find_package(Python3 COMPONENTS Interpreter)
   if(NOT Python3_Interpreter_FOUND)
